inc/TRestRawBaseLineCorrectionProcess.h

```diff
@@ -23,26 +23,33 @@
 #ifndef RESTProc_TRestRawBaseLineCorrectionProcess
 #define RESTProc_TRestRawBaseLineCorrectionProcess
 
-#include "TRestEventProcess.h"
+#include <TRestEventProcess.h>
+
+#include "TRestRawReadoutMetadata.h"
 #include "TRestRawSignalEvent.h"
 
+using namespace std;
```

Comment:
```suggestion
```

This is a really bad idea @JPorron, read https://stackoverflow.com/questions/9764481/should-i-include-header-file-within-a-namespace for example to understand why.